ACM_General/thirdparty_auth/views.py

```diff
@@ -251,6 +255,8 @@ def post_auth(request, cleaned_data):
 
         if user is not None:
             login(request, user)
+            messages.success(request, 'You have been logged in, ' +
```

Comment:
Prefer format `"You have been logged in {}".format(user)` over the string concatination.